packages/frontend-2/components/projects/Dashboard.vue

```diff
@@ -11,10 +11,15 @@
         v-if="projectsPanelResult?.activeUser?.projectInvites?.length"
         :invites="projectsPanelResult?.activeUser"
       />
-      <ProjectsNewSpeckleBanner
-        v-if="showNewSpeckleBanner"
-        @dismissed="onDismissNewSpeckleBanner"
-      />
+      <WorkspaceInviteBanners
+        v-if="projectsPanelResult?.activeUser?.workspaceInvites?.length"
+        :invites="projectsPanelResult.activeUser"
+      >
+        <ProjectsNewSpeckleBanner
```

Comment:
Just checking that you meant to nest ProjectsNewSpeckleBanner in WorkspaceInviteBanner. 

ProjectsNewSpeckleBanner is removed in another branch anyway

Reply:
fixed